apps/web/src/components/dashboard/ExplorePossibleWidget/index.tsx

```diff
@@ -0,0 +1,240 @@
+import { Card, Stack, Typography, Box, Chip, IconButton } from '@mui/material'
+import KeyboardArrowLeftIcon from '@mui/icons-material/KeyboardArrowLeftRounded'
+import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRightRounded'
+import Link from 'next/link'
+import { useState, useRef, useEffect, useMemo } from 'react'
+import { useRouter } from 'next/router'
+import type { UrlObject } from 'url'
+import { AppRoutes } from '@/config/routes'
+import { useTxBuilderApp } from '@/hooks/safe-apps/useTxBuilderApp'
+import { trackEvent } from '@/services/analytics'
+import { EXPLORE_POSSIBLE_EVENTS } from '@/services/analytics/events/overview'
+import { useDarkMode } from '@/hooks/useDarkMode'
+import css from './styles.module.css'
+
+export type ExplorePossibleApp = {
+  id: string
+  title: string
+  iconUrl: string
+  link: string | UrlObject
+  tag?: {
+    text: string
+    color: string
+  }
+}
+
+const ITEM_WIDTH = 180
+const ITEM_GAP = 16
+
+const ExplorePossibleWidget = () => {
+  const router = useRouter()
+  const txBuilderApp = useTxBuilderApp()
+  const isDarkMode = useDarkMode()
+  const [canScrollLeft, setCanScrollLeft] = useState(false)
+  const [canScrollRight, setCanScrollRight] = useState(false)
+  const scrollContainerRef = useRef<HTMLUListElement>(null)
+
+  const EXPLORE_POSSIBLE_APPS: ExplorePossibleApp[] = useMemo(
```

Comment:
Can you define this as a static config outside of the component? Parametrization for the dark mode and router query can be done dynamically when it maps this to JSX.